### PR TITLE
[WIP] Get rid of the imp module usage

### PR DESCRIPTION
**Pull request by slsh1o**
The imp library deprecated since python 3.3 so it was changed to importlib

Close: #602

### CODE REVIEW

File: lib/ansiblelint/utils.py [hunk 1]
```diff
@@ -100,19 +100,14 @@ def ansible_template(basedir, varname, templatevars, **kwargs):
 
 def load_plugins(directory):
     result = []
-    fh = None
 
     for pluginfile in glob.glob(os.path.join(directory, '[A-Za-z]*.py')):
-
```
Comment (webknjaz) [inline lib/ansiblelint/utils.py:106]: Please avoid doing unrelated formatting changes.

File: pytest.ini [hunk 2]
```diff
@@ -24,11 +24,6 @@ doctest_optionflags = ALLOW_UNICODE ELLIPSIS
 filterwarnings =
     error
```
Comment (webknjaz) [inline pytest.ini:26]: Since the deprecation warning is happening in the third-party lib that we still support, we should ignore `DeprecationWarning` but only in that lib, not our own code base:
```suggestion
    # TODO: delete the following ignores once Ansible that we support gets rid of `imp`	
    # Ref: https://github.com/ansible/ansible-lint/pull/734
    ignore:the imp module is deprecated in favour of importlib; see the module's documentation for alternative uses:DeprecationWarning:ansible.plugins.loader
```

Comment (webknjaz) [reply pytest.ini:26]: (alternatively, it could be possible to not use that API from the third party that causes this, but only if it provides an alternate implementation)

File: lib/ansiblelint/utils.py [hunk 3]
```diff
@@ -100,19 +100,14 @@ def ansible_template(basedir, varname, templatevars, **kwargs):
 
 def load_plugins(directory):
     result = []
-    fh = None
 
     for pluginfile in glob.glob(os.path.join(directory, '[A-Za-z]*.py')):
```
Comment (webknjaz) [inline lib/ansiblelint/utils.py:104]:
```suggestion
    for pluginfile in glob.glob(os.path.join(directory, '[A-Za-z]*.py')):

```

**Comment by ssbarnea** [comment]
@webknjaz This got merged while still having a "WIP" title...

**Comment by webknjaz** [comment]
@ssbarnea no, `pytest.ini` needs to ignore things that happen in Ansible, not in ansible-lint.

**Comment by ssbarnea** [comment]
@webknjaz Corrected title, good. I was more worried about the generating release notes with it. Can we try to enable Release Drafter?

**Comment by webknjaz** [comment]
Honestly, I prefer towncrier. Also, this change does not look like it's worth including in the changelog because there's no immediate impact for the end-user.
